lib/kubeclient/common.rb

```diff
@@ -416,6 +428,18 @@ def api
       JSON.parse(response)
     end
 
+    def self.restclient_read_timeout_option
```

Comment:
private?

do you mind putting this method next to our exiting validate_X methods?